src/dockleHelper.ts

```diff
@@ -29,6 +32,25 @@ const TITLE_TITLE = "TITLE";
 const TITLE_SEVERITY = "SEVERITY";
 const TITLE_DESCRIPTION = "DESCRIPTION";
 
+export async function runDockle(): Promise<number> {
+    const docklePath = await getDockle();
+    core.debug(util.format("Dockle executable found at path ", docklePath));

```

Comment:
This debug log can be part of  'getDockle()' function.

Reply:
Sounds good. I'll update that.